.github/ISSUE_TEMPLATE/Standard-Feature-Issue-Template.md

```diff
@@ -22,31 +22,44 @@ so that < why do you need this (short description of the business value >.
 ## Notes
 < bulleted list of special notes or suggestions to the development team on the implementation of the changes. >  
 
-## Definition of Ready
+## Ready for Planning
 
 - [ ] The Issue maintains standard format for Issues and meets INVEST
-- [ ] Issue Defined and written appropriately
-- [ ] Acceptance Criteria Defined
-- [ ] User acceptance test cases defined
+- [ ] Story and Business Value complete
+- [ ] Initial Acceptance Criteria Defined
 - [ ] The person who will Approve story identified
-- [ ] Dependencies Identified
+
+## Ready for Ready to Work
+- [ ] Acceptance Criteria Refined
 - [ ] Issue size estimated by Dev Team
-- [ ] Proposed solution accepted
+- [ ] Dependencies Identified
 
-## Definition of Done
+## Ready for Review
+- [ ] Proposed solution defined and accepted
+- [ ] Code defined, implemented, and built
+- [ ] Unit test defined, implemented, and passed
 
-- [ ] Issue accepted by PO
-- [ ] Issue is scored
+## Ready for Pre-Production QA
+- [ ] Peer Reviews complete with all changes tested
 - [ ] All features implemented
 - [ ] All features not yet implemented or inactive are hidden from the user
-- [ ] Peer Reviews complete	
```

Comment:
I would only add `[] Test cases created and reviewed` which will later function as an input to the automation tasks, it will also help us to start tracking our test coverage.